OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -15,6 +15,8 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public enum ReferencePoints { TopLeft, BottomRight }
```

Comment:
How do you feel about making this field then of type `Tuple<ReferencePointY, ReferencePointX>`? Apart from the FieldLoader addition, it would make for nicer code w.r.t. the conditional branching.